zu_aktualisieren/daten.tex

```diff
@@ -15,15 +15,16 @@
 \newcommand{\prodekanphysikfotoA}{bilder/hcsc_mon.jpg}
 \newcommand{\prodekanphysikB}{U. Schwarz}
 \newcommand{\prodekanphysikfotoB}{bilder/schwarz_mon.jpg}
-\newcommand{\studiendekanphysik}{Norbert Herrmann}
+\newcommand{\studiendekanphysik}{Jörg Jäckel}
 \newcommand{\studiendekanphysikfoto}{bilder/herrmann_mon.jpg}
+	%hier muss ein Foto von Prof. Jäckel eingefügt werden
 \newcommand{\pruefausschussvorsitzphysik}{Cornelis Dullemond}
 \newcommand{\studienberatungphysik}{Ulrich Uwer und Michael Hausmann}
-\newcommand{\dozentvorkurs}{Herrn Thommes, Herrn Jaeckel, Herrn Schäfer und Herrn Komnik}
-\newcommand{\bafogphysik}{Norbert Herrmann}
+\newcommand{\dozentvorkurs}{Herr Thommes und Herr Weigand}
```

Comment:
Es muss "Herrn" mit N am Ende heißen, damit es an der entsprechenden Stelle in den Satzbau passt.